qa/L0_perf_analyzer/test.sh

```diff
@@ -639,6 +639,30 @@ for PROTOCOL in grpc http; do
         fi
         set -e
     done
+
+    # Testing that trace logging works
+    set +e
+    curl localhost:8000/v2/trace/setting -d '{"trace_level":["TIMESTAMPS"]}'
```

Comment:
Shouldn't the `$PERF_ANALYZER ... --trace-level TIMESTAMPS` do this for you?

Reply:
Good catch--fixed